File: Python Code/Daily_Flash_Code_0/Week 8/11 Mar/44-DailyFlash_Solutions/44-DailyFlash_Solutions/11-Mar/Java/prog3.java
import java.util.Scanner;
class Demo {
	public static void main(String[] args) {
		int num = 0, f = 0, fact = 1;
		Scanner sc = new Scanner(System.in);
		do {
			if(f > 0)
				System.out.println(fact);
			num = sc.nextInt();
			fact = 1;
			for(int i = 1; i <= num; i++) {
				fact *= i;
			}		
			f = 1;
		} while(num >= 0);
	
	}
}

File: Python Code/Daily_Flash_Code_0/Week 8/11 Mar/44-DailyFlash_Solutions/44-DailyFlash_Solutions/11-Mar/C++/prog3.cpp
#include<iostream>

int main() {
	int num, f = 0, fact = 1;
	do {
		if(f > 0)
			std::cout << fact << "\n";
		fact = 1;
		std::cin >> num;
		for(int i = 1; i <= num; i++) {
			fact *= i;
		}
		f = 1;
	} while(num >= 0);
	return 0;
}
